source/Calamari.AzureResourceGroup/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly:InternalsVisibleTo("Calamari.AzureResourceGroup.Tests")]
```

Comment:
These should be in a `Properties` folder according to @slewis74

Reply:
Fixed